src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -881,8 +881,8 @@ public int DesiredWidth
             }
             set
             {
-                UpdateImage(ImageVisualProperty.DesiredWidth, new PropertyValue(value));
                 _desired_width = value;
```

Comment:
It maybe better.
if (_desired_width != value)
{
    _desired_width = value;
    UpdateImage(0, null);
}

This is same for the DesiredHeight.